db/migrate/20200121190901_create_account_users.rb

```diff
@@ -0,0 +1,26 @@
+class CreateAccountUsers < ActiveRecord::Migration[6.0]
+  def change
+    create_table :account_users do |t|
+      t.references :account, foreign_key: true, index: true
+      t.references :user, foreign_key: true, index: true
+
+      t.integer :role, default: 0
+      t.bigint :inviter_id
+
+      t.timestamps
+    end
+
+    ::User.all.each do |user|
+      account_user = ::AccountUser.create!(
```

Comment:
If we can do this User fetch in batches, well and good. But this also works. Also before creating the `AccountUser` if we can check whether an `account_user` already exists for the user, that could be really helpful if in case we experience any issues with the migration.

Reply:
makes sense. will add the check